Source/Network/Router/MLCardFormApiRouter.swift

```diff
@@ -49,64 +49,57 @@ enum MLCardFormApiRouter {
+        case .postWebPayFinishInscription(let headers, _):
             return [MLCardFormWebPayService.HeadersKeys.contentType.getKey: headers.contentType,
                     MLCardFormWebPayService.HeadersKeys.xpublic.getKey: headers.xpublic,
-                    MLCardFormBinService.HeadersKeys.xFlowId.getKey: headers.xFlowId]
+                    MLCardFormBinService.HeadersKeys.xFlowId.getKey: headers.xFlowId,
+                    MLCardFormBinService.HeadersKeys.sessionId.getKey: headers.sessionId,
+                    MLCardFormWebPayService.HeadersKeys.accessToken.getKey: headers.accessToken]
         }
     }
 
     var parameters: [URLQueryItem] {
-        switch self {
-        case .getCardData(let queryParams, _):
-            var urlQueryItems = [
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.bin.getKey, value: queryParams.bin),
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.siteId.getKey, value: queryParams.siteId),
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.platform.getKey, value: queryParams.platform),
-                URLQueryItem(name: MLCardFormBinService.QueryKeys.odr.getKey, value: String(queryParams.odr))
-            ]
-            if let excludedPaymentTypes = queryParams.excludedPaymentTypes {
-                urlQueryItems.append(URLQueryItem(name: MLCardFormBinService.QueryKeys.excludedPaymentTypes.getKey, value: excludedPaymentTypes))
-            }
-            return urlQueryItems
-        case.postCardTokenData(let queryParams, _, _):
-            var urlQueryItems:[URLQueryItem] = []
-            if let accessToken = queryParams.accessToken {
-                urlQueryItems.append(URLQueryItem(name: MLCardFormAddCardService.QueryKeys.accessToken.getKey, value: accessToken))
-            } else if let publicKey = queryParams.publicKey {
-                urlQueryItems.append(URLQueryItem(name: MLCardFormAddCardService.QueryKeys.publicKey.getKey, value: publicKey))
+            switch self {
+            case .getCardData(let queryParams, _):
+                var urlQueryItems = [
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.bin.getKey, value: queryParams.bin),
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.siteId.getKey, value: queryParams.siteId),
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.platform.getKey, value: queryParams.platform),
+                    URLQueryItem(name: MLCardFormBinService.QueryKeys.odr.getKey, value: String(queryParams.odr))
+                ]
+                if let excludedPaymentTypes = queryParams.excludedPaymentTypes {
+                    urlQueryItems.append(URLQueryItem(name: MLCardFormBinService.QueryKeys.excludedPaymentTypes.getKey, value: excludedPaymentTypes))
+                }
+                return urlQueryItems
+                case.postCardTokenData( _, _),
+                    .postCardData( _, _),
```

Comment:
Da pra suprimir esses (_, _), você só precisa passar eles se for utilizar algum

Reply:
Olhando aqui....

Reply:
Removi e deu varios erros, acredito que para remover vou precisar deixar tudo como opcional

Reply:
ué...que estranho, mas removeu os `()` também né?

Reply:
feito...agora sim, removido o (), rsrsrssr